docker-compose.yml

```diff
@@ -1,13 +1,32 @@
 version: "3"
 
 services:
-  api:
-    build: .
+  daemon:
+    build: 
+      context: ./docker/daemon
+      dockerfile: ./docker/daemon/Dockerfile
```

Comment:
if there's a Dockerfile in there then all you really need to do is `build: ./docker/daemon`